modules/View.js

```diff
@@ -86,12 +87,26 @@
      * @method render
      */
     render: function() {
-      this.unplug();
```

Comment:
I'd like to open another issue to handle this. I'd like this to be handled separately from this commit if possible.